### ISSUE TITLE
exp.py not working due to K.get_values

### ISSUE THREAD

**Issue by francescacipollini** [singh-ml/fsnet#2]
Hello, 
I tried the FSNet code after reading the related rticle but I found some difficulties in using it. 
That's because, in the tinyLayerE call function the code breaks while calling p= K.get_value(self.logits) because it is not possible to call numpy() on a Tensor object. 
Do have any update to solve this issue? 
Thank you

**Comment by mcnewcp** [comment]
@francescacipollini I'm trying to run this now and ran into the same issue.  Did you ever find a solution?
